src/modules/python/pbs_hooks/PBS_alps_inventory_check.PY

```diff
@@ -155,11 +158,12 @@ def get_apstat_nids(msg):
     # not ship the subprocess module
     apstat_out = os.popen(cmd_apstat)
 
+    pattern = re.compile("(.*?)\s+(.*?)\s+(.*?)\s+(.*?)\s+(.*)")
+

```

Comment:
@bhroam :
 *?, +?, ??
The '*', '+', and '?' qualifiers are all greedy; they match as much text as possible. If the RE <.*> is matched against <a> b <c>, it will match the entire string, and not just <a>. Adding ? after the qualifier makes it perform the match in non-greedy or minimal fashion; as few characters as possible will be matched. Using the RE <.*?> will match only <a>.